js_modules/dagster-ui/packages/ui-core/src/automation/VirtualizedAutomationScheduleRow.tsx

```diff
@@ -133,22 +136,33 @@ export const VirtualizedAutomationScheduleRow = forwardRef(
             </Tooltip>
           </RowCell>
           <RowCell>
+            {/* Left aligned content */}
             <Box flex={{direction: 'row', gap: 8, alignItems: 'flex-start'}}>
-              {scheduleData ? (
-                <Box flex={{direction: 'column', gap: 4}}>
-                  {/* Keyed so that a new switch is always rendered, otherwise it's reused and animates on/off */}
-                  <ScheduleSwitch key={name} repoAddress={repoAddress} schedule={scheduleData} />
-                  {errorDisplay(
-                    scheduleData.scheduleState.status,
-                    scheduleData.scheduleState.runningCount,
-                  )}
-                </Box>
-              ) : (
-                <div style={{width: 30}} />
-              )}
-              <Link to={workspacePathFromAddress(repoAddress, `/schedules/${name}`)}>
-                <MiddleTruncate text={name} />
-              </Link>
+              <Box flex={{grow: 1, gap: 8}}>
+                {scheduleData ? (
+                  <>
+                    <ScheduleSwitch key={name} repoAddress={repoAddress} schedule={scheduleData} />
+                    {errorDisplay(
+                      scheduleData.scheduleState.status,
+                      scheduleData.scheduleState.runningCount,
+                    )}
+                  </>
+                ) : (
+                  <div style={{width: 30}} />
+                )}
+                <Link to={workspacePathFromAddress(repoAddress, `/schedules/${name}`)}>
+                  <MiddleTruncate text={name} />
+                </Link>
+              </Box>
+              {/* Right aligned content */}
+              <Button
+                onClick={() => {
+                  setShowTestTickDialog(true);
+                }}
+                style={{height: '24px', marginTop: '-4px'}} // center button text with content in AutomationRowGrid
+              >
+                Manual tick
+              </Button>
```

Comment:
This looks good to me!  It might be nice to wrap this button in a custom component like `<TestTickButton  name= repoAddress= jobName=/>`, and then push the state and the `EvaluateScheduleDialog` rendering one layer down into that component, just to keep this top layer clean. It looks like there isn't quite enough in common between this implementation and the sensor one for that component to be re-used, though - it'd strictly be for readability.



Sidenote, I'm torn about the vertical alignment -- I think it's good for the button's text to share a baseline with the label to it's left, but I wish it also lined up with this content in the next column. I think it'd almost be better to push the name of the schedule/sensor down vs. pulling this up, but I think it might be splitting hairs.

![image](https://github.com/user-attachments/assets/6a6720d1-50cf-4036-9f71-e267bd9c8383)

Reply:
![image](https://github.com/user-attachments/assets/741af8b9-5189-45eb-ab02-d38e58e62eef)

Yeah I think Button aligned with the next column is better, fixed in latest commit